components/OverveiwComponents/vison.tsx

```diff
@@ -0,0 +1,50 @@
+"use client";
+import React from "react";
+import { Eye } from "lucide-react";
+import Image from "next/image";
+import { useTheme } from "next-themes";
+
+export const Vision = () => {
+  const { theme, setTheme } = useTheme();
+
+  return (
+    <div className="flex flex-col dark:text-secondary text-destructive">
+      <Eye className="w-[3rem] h-[3rem]" />
+      <h1 className="font-sans text-card-foreground text-2xl md:text-3xl lg:text-4xl font-bold py-[1rem]">
+        {" "}
+        Vision
+      </h1>
+      <ul className="list-disc pl-5 md:text-[1.25rem] text-[1rem] md:w-[45rem] lg:w-[50rem] text-foreground space-y-3">
+        <li>
+          Empower Decision-Making: Turn complex energy models into clear,
+          actionable insights for policy and investment.
+        </li>
+        <li>
+          Accelerate Insight Delivery: Fast-track the flow of research data into
+          the hands of decision-makers through visualisations & automation.
+        </li>
+        <li>
+          Champion Open Collaboration: Use open data and open-source tools to
+          foster transparency and cooperation.
+        </li>
```

Comment:
A suggestion for a slight style improvement (feel free to disregard, in case!)

```suggestion
        <li>
          Empower Decision-Making: Transform complex energy models into clear,
          actionable insights for policy and investment.
        </li>
        <li>
          Accelerate Insight Delivery: Fast-track the flow of research data to
          decision-makers through visualisations & automation.
        </li>
        <li>
          Champion Open Collaboration: Leverage open data and open-source 
          tools to foster transparency and cooperation.
        </li>
```